packages/esm-outpatient-app/src/patient-info/appointment-details.component.scss

```diff
@@ -4,8 +4,9 @@
 @import '~carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/type/styles';
 @import '~carbon-components/src/globals/scss/vendor/@carbon/layout/scss/generated/spacing';
 
+// to add #d0e2ff in styleguide
 .widgetCard {
-  border: 1px solid $ui-03;
+  border-top: 1px solid #d0e2ff;
```

Comment:
we should use color variables as provided by carbon

Reply:
what if its not among the colors listed ?

Reply:
This color is ``` $carbon--blue-20 ```

Reply:
updated